recipes/traittypes/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.0.4" %}
+{% set md5 = "6378e67cd60c8f8f13bdae2b2a2ef307" %}
+
+package:
+    name: traittypes
+    version: {{ version }}
+
+source:
+    fn: traittypes-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/t/traittypes/traittypes-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+        - traitlets >=4.2.2
+        - numpy
+
+test:
+    imports:
+        - traittypes
+
+about:
+    home: https://github.com/jupyter-incubator/traittypes
+    license: BSD 3-Clause
+    summary: Trait types for NumPy, SciPy and friends
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - jasongrout
```

Comment:
Would be nice if the indentation here matched everything else. Either 2 or 4 spaces should be fine.